modules/sdl2/sdl.c

```diff
@@ -161,32 +176,32 @@ static int alloc(struct vidisp_st **stp, const struct vidisp *vd,
 }
 
 
-static int display(struct vidisp_st *st, const char *title,
-		   const struct vidframe *frame)
+static int display(struct vidisp_st *st)
```

Comment:
I would suggest to keep the `frame` argument here. 

then you dont have to change all the lines from `frame` to `st->frame`
and hence will make the patch smaller